sources/SessionView.h

```diff
@@ -26,7 +26,6 @@
  */
 
 #import <Cocoa/Cocoa.h>
-#import "iTermFindDriver.h"
```

Comment:
SessionView conforms to the `iTermFindDriverDelegate` protocol declared in this header